tests/integration/webserver_sub.py

```diff
@@ -51,11 +54,61 @@ def redirect_later():
     return flask.redirect('/')
 
 
+@app.after_request
+def log_request(response):
+    request = flask.request
+    template = '127.0.0.1 - - [{date}] "{verb} {path} {http}" {status} -'
+    print(template.format(
+        date=datetime.now().strftime('%d/%b/%Y %H:%M:%S'),
+        verb=request.method,
+        path=request.full_path if request.query_string else request.path,
+        http=request.environ['SERVER_PROTOCOL'],
+        status=response.status_code,
+    ), file=sys.stderr, flush=True)
+    return response
+
+
+class WSGIServer(cherrypy.wsgiserver.CherryPyWSGIServer):
+
+    """A custom WSGIServer that prints a line on stderr when it's ready."""
+
+    # pylint: disable=no-member
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._ready = False
+        self._printed_ready = False
+
+    @property
+    def ready(self):
+        return self._ready
+
+    @ready.setter
+    def ready(self, value):
+        if value and not self._printed_ready:
+            print(' * Running on http://127.0.0.1:{}/ (Press CTRL+C to quit)'
+                  .format(self.bind_addr[1]), file=sys.stderr, flush=True)
+            self._printed_ready = True
+        self._ready = value
+
+
 def main():
+    # pylint: disable=no-member
+    # "Instance of 'WSGIServer' has no 'start' member (no-member)"
+    # "Instance of 'WSGIServer' has no 'stop' member (no-member)"
```

Comment:
I reported this now, so please add a `# WORKAROUND for https://bitbucket.org/logilab/pylint/issues/702/` here and above the other `no-member` disable, so I can easily find those when there's a new release.